src/style.css

```diff
@@ -1,3 +1,77 @@
+html {
+  --header-color: #00acdc;
+}
+
 body {
   margin: 0;
+  max-width: 1200px;
+  min-width: 1024px;
+  font-family: Roboto, sans-serif;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  display: flex;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  position: relative;
+  white-space: nowrap;
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 23px 0;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: var(--header-color);
+  transition: color 0.5s;
+}
+
+.is-active {
+  color: var(--header-color);
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 37px;
```

Comment:
don't hardcode width. It should work correctly for any link

Reply:
not fixed
<img width="488" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/98764641/58cbe16e-6f25-4caa-a114-dca527f1af30">